### DISCUSSION TITLE
ACE FRIES

### DISCUSSION THREAD

**Discussion by Arkad407** [acemod/ACE3#8185] [Q&A]
Hi
I wanted to make MH-6J from CUP mod capable to FRIES, but every time i tried i got information about this helicopter is not capable. Then i tried to use info from https://ace3mod.com/wiki/framework/fastroping-framework.html but i dont know what i should write here, can i get some examples?I mean some examples of working code, wchich make FRIES usable

**Comment by Whigital** [comment]
The MH-6 from both `CUP`and `RHS` already have fastroping gantry's, so no need for the `FRIES` device.

What's needed is config attributes so `ACE Fastroping` understands that the heli is capable of the feature. See this example for the `RHS MELB`.

https://github.com/acemod/ACE3/blob/0c58d8b20b9d68e15c9ab2175b8c89c7f024e175/optionals/compat_rhs_usf3/CfgVehicles.hpp#L94-L99

Or you can just get `CUP ACE3 Compatibility Addon - Vehicles` [here](https://drive.google.com/file/d/1f7naHm5jHVRaMunS3l4dtUbkE3IjB94A/view?usp=sharing), provided by the CUP team which already has everything you need :)
